src/styles/srcery.css

```diff
@@ -2,76 +2,92 @@
 Description: Srcery dark color scheme for highlight.js
 Author: Chen Bin <chen.bin@gmail.com>
 Website: https://srcery-colors.github.io/
-Date: 2020-04-06
+Date: 2021-04-13
 */
 
-.hljs {
-  display: block;
-  overflow-x: auto;
-  padding: 0.5em;
-  background: #1C1B19;
-  color: #FCE8C3;
-}
-
-.hljs-strong,
-.hljs-emphasis {
-  color: #918175;
-}
-
-.hljs-bullet,
+/* Bright White */
 .hljs-quote,
-.hljs-link,
-.hljs-number,
-.hljs-regexp,
 .hljs-literal {
-  color: #FF5C8F;
-}
-
-.hljs-code,
-.hljs-selector-class {
-  color: #68A8E4
+  color: #FCE8C3;
 }
 
-.hljs-emphasis {
-  font-style: italic;
+/* Bright Blue */
+.hljs-type,
+.hljs-symbol {
+  color: #68A8E4;
 }
 
+/* Red */
 .hljs-keyword,
-.hljs-selector-tag,
-.hljs-section,
-.hljs-attribute,
-.hljs-variable {
+.hljs-deletion {
   color: #EF2F27;
 }
 
+/* Yellow */
 .hljs-name,
+.hljs-function,
+.hljs-attribute,
+.hljs-selector-attr,
+.hljs-selector-id,
+.hljs-selector-class,
+.hljs-selector-pseudo,
+.hljs-section,
 .hljs-title {
   color: #FBB829;
 }
 
-.hljs-type,
-.hljs-params {
+/* Cyan */
+.hljs-code,
+.hljs-variable,
+.hljs-property,
+.hljs-template-variable,
+.hljs-class {
   color: #0AAEB3;
 }
 
-.hljs-string {
+/* Bright Green */
+.hljs-string,
+.hljs-regexp,
+.hljs-bullet,
+.hljs-addition {
   color: #98BC37;
 }
 
-.hljs-subst,
+/* Bright Magenta */
 .hljs-built_in,
-.hljs-symbol,
-.hljs-selector-id,
-.hljs-selector-attr,
-.hljs-selector-pseudo,
+.hljs-params {
+  color: #FF5C8F;
+}
+
+/* Blue */
 .hljs-template-tag,
-.hljs-template-variable,
-.hljs-addition {
-  color: #C07ABE;
+.hljs-selector-tag {
+  color: #2C78BF;
 }
 
+/* Bright black */
+.hljs-link,
+.hljs-number,
 .hljs-comment,
-.hljs-deletion,
 .hljs-meta {
   color: #918175;
 }
+
+.hljs,
```

Comment:
This looks incorrect.  `hljs-subst` should perhaps get BG and FG applied, but not different layout attributes.  Should `subst` be split out with just the color and background?

Reply:
@joshgoebel , thanks for the review. I force pushed the latest code.

I didn't do any radical change. I follow the Emacs implementation of same theme and make some small adjustment.